### DISCUSSION TITLE
Obtaining individual post prediction with Pmetrics 2

### DISCUSSION THREAD

**Discussion by uqenovy** [LAPKB/Pmetrics#171] [Q&A] [answered]
Hi everyone

What is the equivalence of the code " plot(mdata.1,pred = post.1, join = F,pch = 3,overlay=F)" (from Pmetrics 1) in the current version of Pmetrics2.

Thank for your help.

EN

**Comment by mnneely** [comment, marked as answer]
All the *plotly* plots in Pmetrics have `line` and `marker` arguments. These are boolean (T/F) for defaults or to suppress, and lists to control details. In this case:

```
run1 <- PM_load(1)
run1$data$plot(line = list(pred = run1$post, join = FALSE), marker = list(symbol = 3), overlay = FALSE)
```
Note that `line` is a list of lists, because there are both prediction lines and joining lines possible when plotting data. On the other hand, there is only one kind of `marker`, so it is a simple list.

As of the most recent versions of R, `T` and `F` are no longer exactly equivalent to `TRUE` and `FALSE`, so I have started writing them out fully.

**Comment by mnneely** [comment]
Hi @uqenovy,

If that answer helped you, would you mind marking it as an answer?

Thanks.
